packages/synapse-react-client/src/components/ReleaseCard/ReleaseCardMedium.tsx

```diff
@@ -84,17 +85,22 @@ export const ReleaseCardMedium: React.FunctionComponent<
         <StatGrid key={stat.label} {...stat} />
       ))}
       <Box gridColumn={SPAN_ALL_COLS} gridRow="span 1" mt="20px" mb="5px">
-        <Link href={releaseCardConfig.requestAccessPath} fontSize="14px">
+        <Button
+          component={ReactRouterLink}
+          to={releaseCardConfig.requestAccessPath}
+          sx={{ fontSize: '14px' }}
+        >
           Request Access
-        </Link>
+        </Button>
```

Comment:
Use react-router-dom's `Link` to prevent a full page reload in the medium card as well